app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -315,10 +351,15 @@ public void onReceiveValue(String errorPage) {
                 });
     }
 
-    private static class CallbackWrapper implements IWebView.Callback {
-        final IWebView.Callback callback;
+    @Override
+    public View getView() {
+        return this;
+    }
+
+    private static class TabViewClientWrapper extends TabViewClient {
```

Comment:
I don't see why we're wrapping the TabViewClient. Can we ignore the wrapping and use TabViewClient directly?